src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -159033,5 +159033,78 @@ class C
                 //                 c1.
                 Diagnostic(ErrorCode.ERR_SemicolonExpected, "").WithLocation(15, 20));
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74297")]
+        public void MissingSystemNullable()
```

Comment:
Nit: should the test be in `NullableTests` instead of `NullableReferenceTypesTests`? It's about nullable value types.

Reply:
Yeah, sounds good, I will move them, thanks.